libs/component-library/src/lib/progress-bar/progressBar.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+import { FxBox, FxBoxProps } from '../box/box';
+import { configureEaseInOutLayoutAnimation } from '../utils/animations';
+import { WINDOW_WIDTH } from '../utils/constants';
+
+export type FxProgressBarProps = {
+  progress: number;
+  width?: number;
+  total?: number;
+  disabled?: boolean;
```

Comment:
`disabled` doesn't look used?

Reply:
Ah yeah I removed that and rebase pulled it back in.